### ISSUE TITLE
6.3 Anrufidentifizierung auf eingehende Chat- oder Messenger-Nachrichten ausweiten

### ISSUE THREAD

**Issue by CarstenKaul** [BIK-BITV/BIK-Web-Test#72]
Diese Anforderung lässt sich nach meinem Verständnis auch auf eingehende Chat- oder Messenger-Nachrichten ausweiten. Falls dies eure Zustimmung findet sollte das in der Prüfschrittbeschreibung mit aufgenommen werden.

**Comment by johannesFischer84** [comment]
Stimmt, in der Anforderung ist neben der "caller identification" auch noch von "similar telecommunications functions" die Rede. Das war mir bisher auch nicht bewusst. Ich denke, dass Chat-/Messenger-Nachrichten von diesem Begriff mitumfasst wären. Ich finde deinen Hinweis gut.

**Comment by CarstenKaul** [comment]
Das habe ich in der Text-Praxis häufig, dass der Sender eingehender Nachrichten nicht unmittelbar identifiziert werden kann und nur über Umwege wahrnehmbar ist.
